README.md

```diff
@@ -11,96 +11,136 @@
 
-Ask for help or report issues:
-* [Chat with the community on Discord](https://discord.gg/f6aerfE)
-* [Discuss topics on GitHub discussions](https://github.com/stride3d/stride/discussions)
-* [Report engine issues](https://github.com/stride3d/stride/issues)
-* [Donate to support the project](https://opencollective.com/stride3d/)
-* [List of Projects made by users](https://github.com/stride3d/stride/wiki/Community-Projects)
-* [Awesome Stride - Huge List of Stride Resources](https://github.com/Doprez/Awesome-Stride)
-* [Localization](https://doc.stride3d.net/latest/en/contributors/engine/localization.html)
+## 🗺️ Roadmap
 
-## Building from source
+Our [Roadmap](https://doc.stride3d.net/latest/en/contributors/roadmap.html) communicates upcoming changes to the Stride engine.
+
+## 🛠️ Building from Source
 
 ### Prerequisites
 
-1. **Latest** [Git](https://git-scm.com/downloads) **with Large File Support** selected in the setup on the components dialog and for convenience a git UI client like [GitExtensions](https://gitextensions.github.io/).
-2. [.NET 8.0 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)
-   - Run `dotnet --info` in a console or powershell window to see which versions you have installed  
-3. [Visual Studio 2022](https://www.visualstudio.com/downloads/) with the following workloads:
-   - `.NET desktop development` with `.NET Framework 4.7.2 targeting pack` (should be enabled by default)
-   - `Desktop development with C++` with
-     - `Windows 10 SDK (10.0.18362.0)` or later version (should be enabled by default)
-     - `MSVC v143 - VS2022 C++ x64/x86 build tools (Latest)` (should be enabled by default)
-     - `C++/CLI support for v143 build tools (Latest)` **(not enabled by default)**
-   - Optional (to target iOS/Android): `.NET Multi-paltform App UI development` and `Android SDK setup` individual component (enabled by default), then in Visual Studio go to `Tools > Android > Android SDK Manager` and install `NDK` (version 20.1+) from `Tools` tab.
-   - Optional (to build VSIX package): `Visual Studio extension development`
+1. **Latest [Git](https://git-scm.com/downloads)** with **Large File Support** selected during setup. For convenience, you might also use a Git UI client like [GitExtensions](https://gitextensions.github.io/).
+2. **[.NET 8.0 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)**
+   - Run `dotnet --info` in a console or PowerShell window to see which versions you have installed.
+3. **[Visual Studio 2022](https://visualstudio.microsoft.com/downloads/)** (the Community edition is free) with the following workloads:
+   - **.NET desktop development** with **.NET Framework 4.7.2 targeting pack** (should be enabled by default)
+   - **Desktop development with C++** with:
+     - **Windows 11 SDK (10.0.22621.0)** or a later version (should be enabled by default)
```

Comment:
Win 11 SDK ? Are you sure about this?

Reply:
100%, tested on clean Win 11. It is selected by default actually by the installer.

Reply:
We had there before Windows 10 SDK

Reply:
I think this requirement applies to UWP, so it's not a big deal. I wonder if version is also correct🤔

Reply:
It is, see below:

<img width="929" alt="image" src="https://github.com/user-attachments/assets/3a36b678-2304-4e9d-8e94-e5a180812309">